src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,112 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private int capacity = DEFAULT_CAPACITY;
+    private Node<K, V> []table = new Node[capacity];
```

Comment:
You should not specify the capacity of the array at the declaration. It's better to initialize it in the constructor. Also, you can use the diamond operator here for a cleaner code: `new Node<>[capacity];`